plugins/JetAnalyzer.cc

```diff
@@ -1158,23 +1198,26 @@ float JetAnalyzer::CalculateHT(const edm::Event& iEvent, const edm::EventSetup&
 
 	//First of all, jet id selections
         // Quality cut
-        if(JetId==3 && !isTightJet(jet)) continue;
-        if(JetId==1 && !isLooseJet(jet)) continue;
-        if(RecalibrateJets) CorrectJet(jet, *rho_handle, PVCollection->size(), isMC);
+        if(JetId==2 && !isTightJet(jet,DataEra)) continue;
+        if(JetId==1 && !isLooseJet(jet,DataEra)) continue;
+	//obsolete
+        ////if(RecalibrateJets) CorrectJet(jet, *rho_handle, PVCollection->size(), isMC);
 
         // JEC Uncertainty
-        if (!isMC){
-            jecUncDATA->setJetEta(jet.eta());
-            jecUncDATA->setJetPt(jet.pt()); // here you must use the CORRECTED jet pt
-        } else {
-            jecUncMC->setJetEta(jet.eta());
-            jecUncMC->setJetPt(jet.pt()); // here you must use the CORRECTED jet pt
-        }
-        if(RecalibrateMass) CorrectMass(jet, *rho_handle, PVCollection->size(), isMC);
+        //if (!isMC){
+        //    jecUncDATA->setJetEta(jet.eta());
+        //    jecUncDATA->setJetPt(jet.pt()); // here you must use the CORRECTED jet pt
+        //} else {
+        //    jecUncMC->setJetEta(jet.eta());
+        //    jecUncMC->setJetPt(jet.pt()); // here you must use the CORRECTED jet pt
+        //}
```

Comment:
Should we really leave JEC out, or do we just need to implement the same method as in `FillJetVector`?

Reply:
JEC are already applied in the config with the `updateJet` method, we don't need to reapply them

Reply:
Right! And we only use them explicitly in the `FillJetVector` in order to store the uncertainty, correct?